### PR TITLE
Update socials

### PR DESCRIPTION
**Pull request by pacorroX**
I just updated the Discord logo and changed everything related to Twitter to X.

### CODE REVIEW

File: components/Community.tsx [hunk 1]
```diff
@@ -21,10 +21,10 @@ const items = [
     icon: <Discord />,
   },
   {
-    name: 'Twitter',
-    description: 'Follow us on Twitter for the latest news and updates.',
-    url: TWITTER_URL,
-    icon: <Twitter />,
+    name: 'X',
+    description: 'Follow us on X for the latest news and updates.',
```
Comment (uncenter) [inline components/Community.tsx:25]: I'm not so sure about going all in on the X rebrand this early... maybe we could do `'X (Twitter)'`?
Ex: `'Follow us on X (Twitter) for the latest news and updates.'`

Just a thought!

Comment (pacorroX) [reply components/Community.tsx:25]: yeah, just saw that Vercel is doing that:
image

**Comment by uncenter** [comment]
image

Hmm... looks weird. I thought it was a typo at first- any ideas?

File: components/Community.tsx [hunk 2]
```diff
@@ -22,7 +22,7 @@ const items = [
   },
   {
     name: 'X',
-    description: 'Follow us on X for the latest news and updates.',
+    description: 'Follow us on X (Formerly Twitter) for the latest news and updates.',
```
Comment (uncenter) [inline components/Community.tsx:25]: Drop the 'Formerly'?

**Comment by pacorroX** [comment]
I think the labels are fine and because we have space in the card we can also have the 'Formerly' too.

**Comment by uncenter** [comment]
> I think the labels are fine and because we have space in the card we can also have the 'Formerly' too.

If you are insistent on keeping it I would at least make it lowercase: `X (formerly Twitter)`.

**Comment by uncenter** [comment]
> I think the labels are fine and because we have space in the card we can also have the 'Formerly' too.

The labels definitely need to change. The logo and the label for X doesn't look right since the logo and the name are identical.

**Comment by pacorroX** [comment]
> > I think the labels are fine and because we have space in the card we can also have the 'Formerly' too.
> 
> The labels definitely need to change. The logo and the label for X doesn't look right since the logo and the name are identical.

And how you think the labels should be?

**Comment by uncenter** [comment]
> > > I think the labels are fine and because we have space in the card we can also have the 'Formerly' too.
> > 
> > 
> > The labels definitely need to change. The logo and the label for X doesn't look right since the logo and the name are identical.
> 
> And how you think the labels should be?

Not sure... but this is for sure not a good look:

image

**Comment by pacorroX** [comment]
> > > > I think the labels are fine and because we have space in the card we can also have the 'Formerly' too.
> > > 
> > > 
> > > The labels definitely need to change. The logo and the label for X doesn't look right since the logo and the name are identical.
> > 
> > 
> > And how you think the labels should be?
> 
> Not sure... but this is for sure not a good look:
> 
> image
> Maybe we could try putting the labels below the logos? Or try removing the label text entirely like I demoed above?

Went back to this and did that. I think is just fine, it is just changing Twitter to X.

**Comment by uncenter** [comment]
The website has changed quite a bit since this PR was opened, you'll need to figure out how to handle those conflicts before we discuss that I think.

**Comment by pacorroX** [comment]
If it isn't worth anynore, I'm closing this it.

**Pull request closed without merging**